src/ml/neural_net/float16.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright © 2020 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#pragma once
+
+#ifdef __APPLE__
```

Comment:
please be consistent (especially within the same file) with `ifdef` vs `if defined`. i tend to prefer `if defined`.

Reply:
I think in our repo, we mostly `#if def`

Reply:
I changed all of them to be `#ifdef`

Reply:
Sorry I didn't change it in that file. Changing it now